.github/workflows/create-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Create Unity Package
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
I added this so we can test in a feature branch we specify, but this workflow has to be merged at least to develop (the default branch) before we can use it.